apps/blog/src/app/global.css

```
@@ -88,3 +88,395 @@
+.agent-prompt-text-ghost {
+  visibility: hidden;
+  display: block;
+  white-space: pre-wrap;
+}
+
+.agent-prompt-caret {
+  display: inline-block;
+  width: 2px;
+  height: 1em;
+  margin-left: 2px;
+  background: currentColor;
```

Stylelint (17.11.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)